app/experimenter/normandy/client.py

```diff
@@ -44,6 +44,12 @@ def get_recipe(recipe_id):
     return recipe_data["approved_revision"]
 
 
+def get_recipe_list(params={}):
```

Comment:
We know this will allways only be called to get the recipes for an experiment slug, so rather than exposing the whole params dict, this function could just take in a slug and build the dict internally.  If you expose the whole raw params dict to callers outside of the normandy module, then they have to know something about how the normandy API works, could inject parameters wrong, etc